sherpa/plot/__init__.py

```diff
@@ -22,12 +22,34 @@
 
 Classes provide access to common plotting tasks, which is done by the
 plotting backend defined in the ``options.plot_pkg`` setting of the
-Sherpa configuration file. Note that the plot objects can be created,
-and used, even when there is no available plot backend, it is just
+Sherpa configuration file. Note that plot objects can be created
+and used even when there is no available plot backend. It is just
 that no graphical display will be created.
 
-"""
+Which backend is used?
+----------------------
+
+When this module is first imported, Sherpa tries to import the
+backends installed with Sherpa in the order listed in
+the ``sherpa.rc`` startup file. The first module that imports
+successfully is set as the active backend. The following command prints the
+name and the location on disk of that module::
```

Comment:
```suggestion
backends installed with Sherpa in the order listed in the
``options.plot_pkg`` setting from the ``sherpa.rc`` startup file. The
first module that imports successfully is set as the active
backend. The following command prints the name and the location on
disk of that module::
```

The copyright year also needs updating but I can't seem to do this via the GH page.

Reply:
Maybe it's finally time that I set up a git pre-commit hook to do that  automatically!